source/class/cv/ui/PageHandler.js

```diff
@@ -84,8 +84,6 @@ qx.Class.define('cv.ui.PageHandler', {
 
       templateEngine.currentPage = pageWidget;
 
-      qx.event.message.Bus.dispatchByName("path."+target+".duringPageChange", target);
```

Comment:
Isn't this a more generic infrastructure to allow widgets and plugins do preperation work to show as soon as the page is available?
=> I guess it should still be in the code